src/AppBundle/Model/SlotCollectionDecorator.php

```diff
@@ -98,6 +98,51 @@ public function getSlotsByType()
         return $slotsByType;
     }
 
+    public static function sortByCycleOrder($s1, $s2)
+    {
+        return intval($s1->getCard()->getCode()) - intval($s2->getCard()->getCode());
+    }
+
+    public static function sortByCycleOrder_deluxeAfter($s1, $s2)
```

Comment:
please use camel-casing throughout, should be `sortByCycleOrderDeluxeAfter`.

Reply:
Solved in 5a40eda332fa1946c3fd2bbf2db53f8c111ae898